modules/pricer/src/main/java/com/opengamma/strata/pricer/cms/SabrExtrapolationReplicationCmsPeriodPricer.java

```diff
@@ -144,7 +144,35 @@ public static SabrExtrapolationReplicationCmsPeriodPricer of(
   public static SabrExtrapolationReplicationCmsPeriodPricer of(double cutOffStrike, double mu) {
     return of(DiscountingSwapProductPricer.DEFAULT, cutOffStrike, mu);
   }
+  
+  /**
+   * Returns the underlying swap pricer.
+   * 
+   * @return the pricer
+   */
+  public DiscountingSwapProductPricer swapPricer() {
```

Comment:
To be consistent with other pricers, these methods should be `getXxx`()` and be located after the constructor

Reply:
Done